addons/web/static/src/js/chrome/systray_menu.js

```diff
@@ -43,11 +37,15 @@ var SystrayMenu = Widget.extend({
         SystrayMenu.Items = _.sortBy(SystrayMenu.Items, function (item) {
             return !_.isUndefined(item.prototype.sequence) ? item.prototype.sequence : 50;
         });
+        var prom = Promise.resolve();
         _.each(SystrayMenu.Items, function (WidgetClass) {
             var cur_systray_item = new WidgetClass(self);
             self.widgets.push(cur_systray_item);
-            self.items.push(cur_systray_item.prependTo(self.$el));
+            prom = prom.then(function () {
+                return cur_systray_item.prependTo(self.$el);
+            });
         });
+        return prom;
```

Comment:
We should start the widgets by happening them in fragments, in parallel (as before), and then prepend them in order in self.$el. That way, all async widgets will be loaded in parallel instead of sequentially (the whole webclient waits for this at startup...)